lib/api/wpapi.js

```diff
@@ -0,0 +1,77 @@
+var $ = require('zepto-browserify').$;
+
+var name = 'WP.CORE';
+
+var baseUrl = 'https://public-api.wordpress.com/wp/v2';
```

Comment:
So this is where the namespace might trip us up a bit.  This baseUrl has all of the default "core" endpoints, but we also have `https://public-api.wordpress.com/wpcom/v2` namespace.  We could potentially have other namespaces down the road too.